beets/metadata_plugins.py

```diff
@@ -251,14 +251,31 @@ def data_source(cls) -> str:
         """
         return cls.__name__.replace("Plugin", "")  # type: ignore[attr-defined]
 
-    def extract_release_id(self, url: str) -> str | None:
+    def _extract_id(self, url: str) -> str | None:
         """Extract an ID from a URL for this metadata source plugin.
 
         Uses the plugin's data source name to determine the ID format and
         extracts the ID from a given URL.
         """
         return extract_release_id(self.data_source, url)
 
+    def _get_id(self, url: str) -> str | None:
+        """Extract an ID from a URL for this metadata source plugin.
+
+        Uses the plugin's data source name to determine the ID format and
+        extracts the ID from a given URL.
+
+        # TODO: Remove this in the future major release, v3.0.0
+        """
+        warnings.warn(
+            "Support for this method is deprecated and will be removed in the future. "
+            "Use _extract_id instead. "
+            "Support for this will be removed in the v3.0.0 release!",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self._extract_id(url)
+
```

Comment:
This method lived for just a couple of weeks and since it's been named to show that it's only used internally, we can do whatever we want with it - thus we can demolish it without any deprecation.